### ISSUE TITLE
Please add additional instruction to build kubicorn from source

### ISSUE THREAD

**Issue by arashkaffamanesh** [kris-nova/kubicorn#77]
Have added a short instruction for newbies to build kubicorn from source:
https://github.com/kubernauts/kubicorn/blob/master/docs/walkthrough.md

This part is new after the line:
$ go get github.com/kris-nova/kubicorn:

If you're new to Go, before you proceed you've to set GOPATH and build the kubicorn binary from source. For this you'd need to build go-bindata and provide it in your path: 

Set GOPATH in your bash profile:

```
$ vi .bash_profil
export GOPATH=/Users/<your user>/go
export PATH=$GOPATH/bin:$PATH
```

```
$ go get github.com/jteeuwen/go-bindata
$ cd $GOPATH/src/github.com/jteeuwen/go-bindata/go-bindata
$ go build
$ cp go-bindata /usr/local/bin/
```
And run make from the src directory of kubicorn:

```
$ cd $GOPATH/src/github.com/kris-nova/kubicorn/
$ make
```
The kubicorn binary will get built and placed under $GOPATH/bin, run kubicorn -h to get the nice Unicorn:

```
$ kubicorn -h
```

The **next** thing

Please add it to the official walkthrough:
https://github.com/kris-nova/kubicorn/blob/master/docs/aws/walkthrough.md

Thanks kindly!

**Comment by kris-nova** [comment]
Hi @arashkaffamanesh can you please open up a pull request with your changes?

**Comment by xmudrii** [comment]
Closed in #121
